vlog4j-examples/pom.xml

```diff
@@ -49,7 +54,16 @@
 			<version>2.7.16</version>
 		</dependency>
 		
-		
+		<dependency>
+			<groupId>fr.lirmm.graphik</groupId>
+			<artifactId>graal-api</artifactId>
+			<version>1.3.1</version>
+		</dependency>
+    	<dependency>
+      		<groupId>fr.lirmm.graphik</groupId>
+      		<artifactId>graal-io-dlgp</artifactId>
+      		<version>1.3.1</version>

```

Comment:
use ${graal.version} property from parent pom

Reply:
Moved version number to parent pom and rewrote examples and graal pom accordingly.